### ISSUE TITLE
Accessibility rating for Search Page can be improved

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/FreeCENMigration#750]
The cen search page is almost identical except that lighthouse has an accessibility rating of 96 (a distinct improvement) and is flagging

Link text (and alternate text for images, when used as links) that is discernible, unique, and focusable improves the navigation experience for screen reader users. Learn more.
Failing Elements the a elements associated with the bottom adrvertisement

E0 U4  S5 C6: P15

**Comment by richpomfret** [comment]
@PatReynolds to do a bit of research here on accessible adverts..

**Comment by PatReynolds** [comment]
Not sure what the "a elements" are (anchor elements?)

**Comment by AlOneill** [comment]
@PatReynolds the `a` element is the link element. (They were called anchors once, when they also had a `name` attribute, but anchor is no longer helpful or accurate.)

**Comment by DeniseColbert** [comment]
Advert no longer there so closing
